.github/release.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright Contributors to the Open Cluster Management project
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+name: Create Release
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Optional:
```suggestion
        uses: actions/checkout@v3
```

Reply:
There is no `v3` for `checkout`: https://github.com/actions/checkout/releases

Reply:
@dhaiducek there is, but v2 is also supported still:
https://github.com/actions/checkout/releases/tag/v3.1.0